apps/courses/src/components/CourseProgressCard.tsx

```diff
@@ -36,11 +36,17 @@ export default ({ content, ...props }) => {
   } = content;
 
   const stats = getCourseProgress(progress, lessons, project?.parts);
-  const percentComplete =
+  let mentorState = props.ms;
```

Comment:
Here, `mentorState` could be a prop instead of `ms` — obtained via destructuring the props. Also, one small suggestion is to improve the variable name. Maybe `isMentor` or `userIsMentor` makes more sense in this context than `mentorState` and `ms`.

Reply:
`const` instead of `let` as it is never reassigned